arch/bootstrap/base.py

```diff
@@ -501,6 +501,10 @@ def conf_int(self, func, reps=1000, method='basic', size=0.95, tail='two',
                 else:
                     a = 0.0
 
+                # TODO: save these parameters?
```

Comment:
What are your thoughts here?

Reply:
Ideally this part ofthe code would be lited into a function taht returns a and b.  That would allow a and b to be unit tested without appending them onto the class, where they don't really belong.  So that I am proposing is to take 487-500 and make a function that computes the a from the bca.  Then take the remaining from 481--510 and produce a second function.  

Does this make sense?

Reply:
Yes, it makes sense. I'll add a commit for this soon.